code_samples/ai_actions/src/Command/AddMissingAltTextCommand.php

```diff
@@ -126,11 +127,13 @@ private function getSuggestedAltText(string $imageEncodedInBase64, string $langu
 
     private function convertImageToBase64(?string $uri): string
```

Comment:
Test `$uri` isn't null earlier, maybe in `shouldGenerateAltText()` or `execute()`.
```suggestion
    private function convertImageToBase64(string $uri): string
```